platforms/software/dependency-management/src/main/java/org/gradle/api/internal/runtimeshaded/RuntimeShadedJarCreator.java

```diff
@@ -230,19 +230,19 @@ private byte[] remapClass(String className, byte[] bytes) {
 
     private static class ShadingClassRemapper extends ClassRemapper {
         final Map<String, String> remappedClassLiterals;
-        private final ImplementationDependencyRelocator remapper;
+        private final ImplementationDependencyRelocator dependencyRelocator;
```

Comment:
Why this rename?

Reply:
There is a `remapper` member in `org.objectweb.asm.commons.ClassRemapper`.
This avoids hiding the field.